Api/Controllers/PostController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Api.DTO;
+using Api.DTO.Events;
+using Api.DTO.Posts;
+using Api.DTO.Users;
+using Domain.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Api.Controllers;
+
+[Route("posts")]
+[ApiController]
+public class PostController : ControllerBase
+{
+    private IPostService postService;
+    private IEventService eventService;
+
+    public PostController(IPostService postService, IEventService eventService)
+    {
+        this.postService = postService;
+        this.eventService = eventService;
+    }
+
+    [HttpGet]
+    [Authorize]
+    [Route("user")]
+    [SwaggerOperation("List user's subscribed events' posts")]
+    public ActionResult<Paged<PostDto>> GetUserEventsPosts([FromQuery] Paging paging, Guid userGuid, bool showAlsoInterested)
+    {
+        var posts = postService.AsUser(User.GetGuid()).GetUsersPosts(userGuid);
+        if (!showAlsoInterested)
+            posts = posts.FindAll(p => p.Event.Participants.FirstOrDefault(part => part.Guid == userGuid) is not null);
+        return Paged<PostDto>.PageFrom(posts.Select(p => p.ToDto()), CreationDateComparer.Instance, paging);
+    }
```

Comment:
We don't need to pass userGuid to GetUsers as user is already passed in in AsUserMethod. But if it was intended for users to see other users subscribed posts then it is ok (I think it wasn't?).